ux.symfony.com/templates/components/RecipeList.html.twig

```diff
@@ -0,0 +1,9 @@
+{% for recipe in this.getRecipes %}
+    <twig:Card
+            name='{{ recipe.name }}'
+            description='{{ recipe.description }}'
+            image='{{ recipe.image }}'
```

Comment:
suggestion(if-minor): Maybe we could have the demo and recipe share fields naming and allow to have only one prop on the Card component.